app/controllers/api/v1/service_offerings_controller.rb

```diff
@@ -17,24 +20,38 @@ def applied_inventories
       private
 
       def send_message_and_generate_task_for(operation_type)
+        logger.info("#{request_id_for_logger}: Entering method send_message_and_generate_task_for operation #{operation_type}")
```

Comment:
I don't think the request id should be in the message.  If it isn't already, it should be set as the [threads current request request_id](https://github.com/ManageIQ/manageiq-loggers/blob/2e83fddf04ae142edc0727b37bf9d10627bc7278/lib/manageiq/loggers/container.rb#L66) so that [the formatter](https://github.com/ManageIQ/manageiq-loggers/blob/2e83fddf04ae142edc0727b37bf9d10627bc7278/lib/manageiq/loggers/container.rb#L47) outputs it and it is searchable in Kibana.

Reply:
yea, I was not sure how Kibana can handle it and how it is usable for searches when we are searching in serialised hashes. so thanks for the clarification 👍